player.js

```diff
@@ -0,0 +1,96 @@
+const LainPlayer = (() => {
+    const playerHTML = "<a id='play-toggle' onclick='LainPlayer.play()'><i class='fa fa-play' aria-hidden='true'></i></a><i id=volume-button class='fa fa-volume-up' aria-hidden='true' onclick='LainPlayer.toggleMute()'></i><input type='range' value=0.51 min=0 max=1  step=0.01 oninput='LainPlayer.changeVolume(this.value)'/>"
```

Comment:
The current conditional stuff uses the `withscript` and `noscript` classes, is there some reason to prefer including the HTML in the javascript here?